src/sql/workbench/contrib/jobManagement/browser/notebooksView.component.ts

```diff
@@ -407,14 +409,14 @@ export class NotebooksViewComponent extends JobManagementView implements OnInit,
 					break;
 				}
 			}
-			this.openLastNRun(targetNotebook, barId, 5);
+			this.openLastNRun(targetNotebook, barId, 5).catch(e => this._logService.error(e));
```

Comment:
Any reason not to use onUnexpectedError instead?

Reply:
From where?

Reply:
https://github.com/microsoft/azuredatastudio/blob/master/src/vs/base/common/errors.ts#L77

Reply:
Didn't know this was a thing, I guess that's a good idea